.devcontainer/README.md

```diff
@@ -0,0 +1,11 @@
+# Development inside a Docker container with Visual Studio Code
+
+```bash
+docker build --force-rm --tag tec-radar/dev:1 .
+
+docker create --name tec-radar_dev --publish 8080:8080 --mount  type=bind,source="C:/Users/marpah/Desktop/Maryam/LTU/ML-Dev/build-your-own-radar/",target=/workspace tec-radar/dev:1
```

Comment:
Please make the command generic and write a little documentation around it, e.g. so that it would also be easy for Sana or Homan to use it.